### PR TITLE
Adjust ui-query protocol

### PR DESCRIPTION
**Pull request by wu-sheng**

### CODE REVIEW

**Comment by wu-sheng** [comment]
If we are using `[Node!]!` as you expected, a lot of parts in protocol need to be changed by following the same style. I hope to catch up in next day.

**Comment by wu-sheng** [comment]
New protocol has been provided:
1. Add set/get `DataTTLConfigs` and set/get `AlarmThreshold` service.
1. All empty arrays are represented by `[]`, not `null`.
1. Add Mutation type, represents for all modification operations.
1. Support **Pagination** in some query.
1. Seperate alarm into to title and content

@peng-yongsheng @hanahmily

**Comment by wu-sheng** [comment]
Make the user node as an extend of Node, rather then an attribute of ApplicationNode.

File: apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/application-layer.graphqls [hunk 1]
```diff
@@ -19,9 +21,6 @@ type ApplicationNode implements Node {
     numOfServerAlarm: Int!
```
Comment (hanahmily) [inline apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/application-layer.graphqls]: Adding 'isAlarm' property that represents application alarm.

Comment (wu-sheng) [reply apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/application-layer.graphqls]: @hanahmily Done.

**Comment by wu-sheng** [comment]
@hanahmily @peng-yongsheng I am asking for further review. Fix all your requirements.

**Pull request merged**